Month1/Week2/Steam/Steam/Game.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <iostream>
+#include <string>
+
+class FGame
+{
+
+private:
+
+	std::string Name;
+	std::string StudioName;
+
+	int Year;
+	int Month;
+	int Day;
+
+public:
+
+	FGame();
+
+	FGame(std::string pName, std::string pStudioName, int pYear, int pMonth, int pDay);
```

Comment:
Yep! The `p` prefix is definitely not a standard we're following and should not be used since introducing the conventions. For constructor parameters I like to use the prefix `In` if it conflicts with the member names (I use this instead of new since technically they are not _new_ values when it's the constructor values.

For setters I do like using New, but `p` should not be used for our program.